contrib/buildrpm/buildrpmLibfabric.sh

```diff
@@ -224,16 +248,33 @@ if [[ -z "$tardirname" ]]; then
 fi
 verbose "Name of package: $tardirname"
 
-version=$(basename "$tardirname" | cut -d- -f2)
-if [[ -z "$tardirname" ]]; then
-  error 4 "Cannot determine version from $tarball"
+if [[ -z "$version" ]] ; then
+  version=$(basename "$tardirname" | cut -d- -f2)
+  if [[ -z "$version" ]]; then
+    error 4 "Cannot determine version from $tarball"
+  fi
 fi
 verbose "Version: $version"
 
 if [[ -n "$install_in_opt" ]]; then
-  prefix="/opt/libfabric/$version"
-  rpmbuild_options="$rpmbuild_options --define '_prefix $prefix'"
+  if [[ -z "$prefix" ]] ; then
+    prefix="$default_opt_prefix"
```

Comment:
Fun fact (not really that important, but a fun fact nonetheless): in shell scripting, if you assign one variable to another, no quotes are needed (even if the RHS contains spaces).  The shell is smart enough to know "oh, this is a straight assignment with no additional string -- I'll copy the entire value, regardless of its contents.

Specifically: even this works:

```sh
$ cat > foo.sh <<EOF
#!/usr/bin/env sh

a="Hello, world"
b=$a
echo b is: $b
EOF
$ chmod +x foo.sh
$ ./foo.sh
b is: Hello, world
```

Reply:
Good stuff. Thanks for the review. Changes put back up. I took each comment, so it should look very similar.